src/main/resources/application.properties

```diff
@@ -0,0 +1,14 @@
+##if you are launch the program for the first time, replace it with "create"
```

Comment:
Not necessary
"update" will create a data base even if it is not created,
if it is created it will be updated without deleting the old data